website/source/docs/commands/namespace.html.md.erb

```diff
@@ -35,37 +35,51 @@ Usage: consul namespace <subcommand> [options] [args]
   ...
 
 Subcommands:
+    create    Create a Namespace
     delete    Delete a Namespace
     list      List all Namespaces
     read      Read a Namespace
-    write     Create or update a Namespace
+    update    Update a Namespace
+    write     Create or update a Namespace from its full definition
 ```
 
 For more information, examples, and usage about a subcommand, click on the name 
 of the subcommand in the sidebar.
 
 ## Basic Examples
 
-Create or update a Namespace:
+Create a Namespace
 
 ```sh
+$ consul namespace create -name team1
+```
+
+  Create or Update a Namespace from its full definition:
+
+  ```sh
 $ consul namespace write ns1.hcl
 ```
 
-Read a Namespace:
+  Read a Namespace:
 
-```sh
+  ```sh
 $ consul namespace read ns1
 ```
 
-List all Namespaces:
+  List all Namespaces:
 
-```sh
+  ```sh
 $ consul namespace list
 ```
 
-Delete a Namespace:
+  Update a namespace
 
-```sh
-$ consul namespace delete ns1
+  ```sh
+$ consul namespace update -name team1 -description "first namespace"
 ```
+
+  Delete a Namespace:
+
+  ```sh
+$ consul namespace delete ns1
```

Comment:
Maybe use `team1` in this example as well?

Reply:
Done